css/src/components/banner.scss

```diff
@@ -22,6 +22,12 @@ $banner-border-width: $border-width !default;
 	border-block: $banner-border-width solid $banner-border-color;
 	line-height: $line-height-normal;
 
+	&[data-dismissable] {
+		display: flex !important;
```

Comment:
Could we avoid `!important` here by reordering? It would be nice to keep `!important` reserved for atomics.

Reply:
Ooops it shouldn't be there in the first place, removed!